NitroxModel/MultiplayerSession/RandomColorGenerator.cs

```diff
@@ -4,9 +4,11 @@ namespace NitroxModel.MultiplayerSession
 {
     public static class RandomColorGenerator
     {
+        private static readonly System.Random random = new System.Random();
+
         public static Color GenerateColor()
         {
-            return new Color32((byte)Random.Range(0, 255), (byte)Random.Range(0, 255), (byte)Random.Range(0, 255), 255);
+            return new Color32(random.NextByte(), random.NextByte(), random.NextByte(), 255);
```

Comment:
This needs to take the appropriate range into account. I think it is 0, 256

Reply:
@MadMaxOfYore That is incorrect. A byte consists of 8 bits, with which it can represent a total of `256` distinct values; from `0` (inclusive) to `255` (inclusive). `NextByte` is an extension method implemented as:
```cs
public static byte NextByte(this System.Random rng) => return Convert.ToByte(rng.Next(256));
```
Where `256` is the **exclusive** `maxValue`.

Any value above `255` would simply wrap around because the upper bytes are discarded.

Reply:
[Random.NextBytes(byte[] buffer)](https://msdn.microsoft.com/en-us/library/system.random.nextbytes(v=vs.110).aspx) should also work so you don't waste data with `Convert.ToByte`.

Reply:
.NET Random implements an exclusive upper bound - so a `Next` call would absolutely be 0, 256

However; I did not notice that the method was `NextBytes` and not `Next` - my mistake ;)